docs/resource-cleanup.md

```diff
@@ -62,36 +69,43 @@ a. remove rbd image(csi-vol-omapval, the prefix csi-vol is value of [volumeNameP
 
 b. remove cephfs subvolume(csi-vol-omapval)
 
-  `[$]ceph fs subvolume rm volume_name subvolume_name group_name`
+  ```text
+  ceph fs subvolume rm volume_name subvolume_name group_name
+  ```
 
   ```bash
-  $ ceph fs subvolume rm  cephfs csi-vol-340daf84-5e8f-11ea-8560-6e87b41d7a6e csi
+  ceph fs subvolume rm  cephfs csi-vol-340daf84-5e8f-11ea-8560-6e87b41d7a6e csi
   ```
 
 ### 4. Delete omap object and omapkey
 
 a. delete omap object
 
-  `[$]rados rm csi.volume.omapval -p pool_name`
+  ```text
+  rados rm csi.volume.omapval -p pool_name
+  ```
 
   ```bash
-  $ rados rm csi.volume.dd2473d0-6a8c-11ea-9113-0ad59d995ce7 -p kube_csi
+  rados rm csi.volume.dd2473d0-6a8c-11ea-9113-0ad59d995ce7 -p kube_csi
   ```
 
 b. delete omapkey
 
-  `[$]rados rmomapkey csi.volumes.default csi.volume.omapkey -p pool_name`
-
+  ```text
```

Comment:
all commands below/above are `bash`, this one moved to `text`

Reply:
Because this is the dummy comment and the one following is the example.

Reply:
That's fine, but should `text` not be mentioned in the guidelines?

Reply:
@nixpanic 
I would rather prefer to remove all `text` uses, because as you mentioned this is not part of the latest Markdown Rules anyway

\```text
My comments
\```

to something simple, like

\```
My comments
\```

Reply:
Yeah, I do not think there is a need to mark sections with `text`. It is not much of a problem, but it is cleaner to drop it.

Reply:
:+1: Will update the PR